### PR TITLE
[147879] Add docs for :bypass_job_queue option in AsyncJob

### PR DESCRIPTION
**Pull request by s417-lama**
Docs for https://github.com/access-company/antikythera/pull/24

https://acsmine.tok.access-company.com/redmine/issues/147879

### CODE REVIEW

**Comment by skirino** [comment]
OK!

File: lib/async_job/async_job.ex [hunk 1]
```diff
@@ -112,6 +112,12 @@ defmodule Antikythera.AsyncJob do
           - 4th attempt (3rd retry) after failure of 3rd attempt is started in `40`s,
           - and so on.
       - If you want to set constant interval, specify `1.0` to second element.
+  - `bypass_job_queue`: Whether or not to run the job immediately, skipping addition to the job queue.
+    Defaults to `false`.
+    If all processes in the executor pool are busy, `register/3` returns `{:error, :no_available_workers}`.
+    In that case, you should call `register/3` again without `bypass_job_queue` option if you really have to run the job.
+    Execution with `bypass_job_queue` option takes advantage of being free from rate limiting (see below) and having no overhead with pushing to the distributed job queue.
+    Please note that this option cannot be used with `schedule`, `attempts` or `retry_interval` because of bypassing the job queue.
```
[the remark this answers is not in the document]
Comment (s417-lama) [reply lib/async_job/async_job.ex]: It looks nice!

Nested lists like that was not rendered correctly, so I fixed a little.

5dab8fde5e225d6b1040b5f548fd85f464920ff7

**Comment by ymtszw** [comment]
# okay 🚀

**Comment by ttokit** [comment]
LGTM!

**Pull request merged**